app/views/devise/registrations/new.html.erb

```diff
@@ -1,7 +1,6 @@
 <div class="row text-center login-screen" style="vertical-align:middle;">
   <div class="col-md-6 col-md-offset-2 logo-image text-center">
-    <%= image_tag @current_organization&.logo_url.present? ? @current_organization&.logo_url : "logo.png",
-                  width: 300, alt: 'mutual-aid-app logo'%>
+    <%= render 'shared/organization_logo' %>
```

Comment:
why would this go in /shared and not under layouts?

Reply:
I was going to ask about this also, but from the other side :).

The convention (?) i've seen is that partials shared between multiple pages go under `view/shared`. I haven't actually seen partials created under `layouts/` unless they are specifically only for use by top level layouts.